internal/woc_client/woc_client.go

```diff
@@ -276,3 +286,61 @@ func (w *WocClient) TopUp(ctx context.Context, mainnet bool, address string) err
 
 	return nil
 }
+
+func (w *WocClient) GetRawTxs(ctx context.Context, mainnet bool, ids []string) ([]*wocRawTx, error) {
+	const maxIDsNum = 20 // value from doc
+
+	var result []*wocRawTx
+
+	net := "test"
+	if mainnet {
+		net = "main"
+	}
+
+	for len(ids) > 0 {
+		bsize := min(maxIDsNum, len(ids))
+		batch := ids[:bsize]
+
+		txs, err := w.getRawTxs(ctx, net, batch)
+		if err != nil {
+			return nil, err
+		}
+
+		result = append(result, txs...)
+		ids = ids[bsize:] // move batch window
+	}
+
+	return result, nil
+}
+
+func (w *WocClient) getRawTxs(ctx context.Context, net string, batch []string) ([]*wocRawTx, error) {
+	payload := map[string][]string{"txs": batch}
```

Comment:
The payload has key "txids"

```
payload := map[string][]string{"txids": batch}
```

https://docs.taal.com/core-products/whatsonchain/transaction#bulk-raw-transaction-data

Reply:
Fixed